src/classes/Recipe.js

```diff
@@ -32,7 +32,7 @@ class Recipe {
               });
             return total
           }, 0);
-            return this.totalCost;
+            return this.totalCost.toFixed(2);
```

Comment:
I've never heard of that so thanks for introducing that to me!

Reply:
Same! I'll have to look this up later. Great fix!